rules/integrations/aws/ml_cloudtrail_error_message_spike.toml

```diff
@@ -23,6 +23,27 @@ interval = "15m"
 license = "Elastic License v2"
 machine_learning_job_id = "high_distinct_count_error_message"
 name = "Spike in AWS Error Messages"
+setup = """## Setup
```

Comment:
The note field, also has existing Setup information and the Unit tests are failing as there are 2 setup informations in Rule.  Please adjust these and remove the already existing setup information.

Reply:
Please check this across all rules, and also update updated_date filed to reflect rule modification date.